quil-rs/src/instruction/calibration.rs

```diff
@@ -20,46 +20,42 @@ pub trait CalibrationSignature {
 
 #[derive(Clone, Debug, Default, PartialEq)]
 pub struct Calibration {
```

Comment:
@MarquessV is open to renaming and breaking this, as long as this PR is already breaking